authors/apps/articles/tests/test_comments.py

```diff
@@ -0,0 +1,173 @@
+            "body": "Woow! Great work"
+            }}
+
+        self.new_comment = {
+            "body": "Good Work"
+            }
+        self.update_comment = {
+            "body": "This is super cool"
+            }
+        self.empty_comment = {
+            "body": ""
+            }
+        self.create_article_data = {
+            "title": "my story",
+            "body": "This is my story",
+            "description": "Here is my story",
+            "images": None,
+            "tagList": ["dragons", "training"]
+
+        }
+
+    def register(self):
+        """Sign up user and get token"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data,
+                                    format='json')
+
+        token = register.data["token"]
+        return token
+
+    def create_article(self):
+        """Create article"""
+        token = self.register()
+        del self.create_article_data['images']
+        response = self.client.post(self.article_listcreate,
+                                    self.create_article_data,
+                                    format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token)
+                                    )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        return response
+
+    def test_update_comment(self):
+        """Test Update Comment"""
+        token = self.register()
+        del self.create_article_data['images']
+        response = self.client.post(self.article_listcreate,
+                                    self.create_article_data,
+                                    format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token)
+                                    )
+        response = self.client.post(reverse('articles:create_comments',
+                                    kwargs={'slug': 'my-story'}),
+                                    self.new_comment,
+                                    format='json',
+                                    HTTP_AUTHORIZATION='token {}'.format(token)
+                                    )
+        id = response.data['id']
+        response = self.client.put(reverse("articles:get_comments",
+                                   kwargs={'slug': 'my-story', "id": id}),
+                                   self.update_comment,
+                                   format='json',
+                                   HTTP_AUTHORIZATION='token {}'.format(token))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['body'], self.update_comment['body'])
+
+    def test_reply_to_comment(self):
+        """Test Update Comment"""
+
+        token = self.register()
+        """Create an article"""
+        del self.create_article_data['images']
+        response = self.client.post(self.article_listcreate,
```

Comment:
I thought you already have a function for this

Reply:
I'll get rid of the function since am not using it.

Reply:
You are using the function in other places... just use it here